Systems/c172p-hydrodynamics.xml

```diff
@@ -275,10 +275,10 @@
       </max>
      </product>
      <max>
-      <value>-5.0</value>
+      <value>-0.0</value>
       <min>
        <property>hydro/transverse-wave/squat-ft</property>
-       <value>5.0</value>
+       <value>0.0</value>
```

Comment:
Yes. Can I write the commit in the merge request or do I need to add one commit for the comment?